source/administration/upgrade.rst

```diff
@@ -31,7 +31,7 @@ Owner and group of the install directory - *{owner}* and *{group}*
 Important Notices
 -----------------
 
-1. Security related changes were made in 3.9.0 that cause any previously created team invite links, password reset links, and email verification links to no longer work. You must update any place where you have published these links.
+1. Security related changes were made in 3.9.0 that cause any previously created email invite links, password reset links, and email verification links to no longer work. Email invites that were sent before the upgrade need to be resent if the recipients have not already signed up to your Mattermost instance. Team invite links are not affected and are still valid.
```

Comment:
Only caution I have is that email invite links are also team invite links because you can send an email invite to a team to an existing user. Not sure it that's an important distinction. Team invites sent by email are invalid, team invites from the link grabbed from the main menu drop down are still valid